pet-store-react-hooks/src/Home/Home.tsx

```diff
@@ -0,0 +1,70 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { getAllPetsAsync, getPetKindsAsync } from '~infrastructure/api';
+import type { Pet } from '~infrastructure/api-types';
+import { ErrorMessage } from '~infrastructure/components/errorMessage/ErrorMessage';
+import { LoadingIndicator } from '~infrastructure/components/loadingIndicator/LoadingIndicator';
+
+import { PetsTable } from './PetsTable';
+
+import './home.css';
+
+export function Home() {
+  const [loading, setLoading] = useState<boolean>(false);
+  const [error, setError] = useState<string>('');
```

Comment:
Use `whatever | undefined` for fetch related `useState`'s. Don't use default values like `[]` or empty strings.